src/Homie/Datatypes/Interface.hpp

```diff
@@ -37,6 +37,7 @@ struct Interface {
   } reset;
 
   GlobalInputHandler globalInputHandler;
+  BroadcastHandler broadcastHandler;
```

Comment:
Seems I missed to add some changes for commit. 

I'm on vacation again, so I'll fix after October 20th.